generated/nidaqmx/_task_modules/triggering/start_trigger.py

```diff
@@ -1037,6 +1037,20 @@ def cfg_dig_pattern_start_trig(
         self._interpreter.cfg_dig_pattern_start_trig(
             self._handle, trigger_source, trigger_pattern, trigger_when.value)
 
+    def cfg_time_start_trig(self, when, timescale=Timescale.USE_HOST):
+        """
+        New Start Trigger
+
+        Args:
+            when (nidaqmx.constants.DateTime): Specifies when to
```

Comment:
`nidaqmx.constants.DateTime` isn't right, it should be either a datetime or hightime

Reply:
Addressed this comment in a separate PR since this needs changing in grpc-device-scrapigen
https://github.com/ni/grpc-device-scrapigen/pull/213